python/MANIFEST.in

```diff
@@ -0,0 +1,9 @@
+include ray/core/src/common/thirdparty/redis/src/redis-server
+include ray/core/src/common/redis_module/libray_redis_module.so
+include ray/core/src/plasma/plasma_store
+include ray/core/src/plasma/plasma_manager
+include ray/core/src/plasma/libplasma.so
+include ray/core/src/local_scheduler/local_scheduler
+include ray/core/src/local_scheduler/liblocal_scheduler_library.so
+include ray/core/src/numbuf/libnumbuf.so
+include ray/core/src/global_scheduler/global_scheduler
```

Comment:
I'm not sure if this MANIFEST.in file serves a purpose anymore because after the last few changes, we are just manually copying all of these files.